backend/authentication/password_validators.py

```diff
@@ -24,3 +25,15 @@ def get_help_text(self):
             "Your password must contain at least 1 numeric character and 1 alphabetic "
             "character"
         )
+
+
+class PasswordStrengthValidator:
+    def validate(self, password, user=None, resultLabel, result):
+        result = zxcvbn(password) resultLabel = "Score: \(result?.score ?? 0)\n"
+        return resultLabel
+
+    def get_help_text(self):
+        return _(
+            "Your password must contain at least 1 numeric character and 1 alphabetic "
```

Comment:
this is the message that will print out when something goes wrong, and this currently doesn't match what we're doing. something like `_("Your password needs to be strong.")` would be good. you could also add in hints, like `_("Your password needs to be strong. Try making it longer or adding in numeric and special characters to make it stronger.")` if you want bonus helpful points :)